numerous/engine/model/model.py

```diff
@@ -415,6 +427,8 @@ def lower_model_codegen(self, tmp_vars):
         self.state_idx, self.derivatives_idx = \
             eq_gen.generate_equations()
 
+        self.logged_variables = {}
```

Comment:
this line should go to __init__

Reply:
it's been moved to assemble. Putting it in __init__ would make it inconsistent with the rest of the code.